src/size_label_prs.py

```diff
@@ -20,10 +22,21 @@ def size_label_prs(data, pull):
     elif additions < 500:
         label = "Size/XL"
 
+    else:
+        label = "Size/XXL"
+        comment = "Please try to break up this PR, it is very large."
+
+    if not comment:
+        comment = label
```

Comment:
We need to add comment only if > 500

Reply:
this was temporary, not intended to stay.